scripts/fetch-release-changelog.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as fs from 'fs';
+import * as process from 'process';
+import * as https from 'https';
+
+/**
+ * Take the `pullRequestNumber` output of the `changesets/action` GitHub action
+ * on standard input and return the CHANGELOG from the PR description in stdout.
+ *
+ * https://github.com/changesets/action#outputs
+ */
+
+const pullRequestNumber = fs.readFileSync(process.stdin.fd, 'utf-8').trim();
+if (!pullRequestNumber || isNaN(parseInt(pullRequestNumber, 10))) {
+  console.error(`No pullRequestNumber from changeset action.`);
+  process.exit(1);
+}
```

Comment:
In the very unlikely event that we enter this block, the release-image would be called with a path with content. I think that would generate an empty image?

I can't think of it ever happening if this is used properly only ever in this workflow so I don't know if we need to really guard for that.